### DISCUSSION TITLE
Potential substack features

### DISCUSSION THREAD

**Discussion by leewynne** [forem/forem#16247] [Feature Requests]
Substack is gaining a lot of moment at the moment for a place to start a newsletter / blog behind a subscription paywall that can either be used for revenue or donations etc. I feel there is definitely scope to allow something similar in Forem for organisations or members that want to produce and deliver higher-value content like video tutorials or something similar.

**Comment by dylanjha** [comment]
Agree that native video would be a great feature add for Forem. A lot of developer content these days is posted in the form of video tutorials in other places.
